Gemfile

```diff
@@ -27,18 +27,25 @@ gem 'puma', '~> 3.0'
 
 gem 'devise-jwt', '~> 0.5.9'
 
+group :test do
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'rspec-rails'
```

Comment:
general question if any of these gems should be in development? I see in the rspec-rails doc it should be on the dev test block (https://github.com/rspec/rspec-rails)

Reply:
ooh good call, I'll move `rspec-rails` to development. I'll also move `faker` to development since we could use it for `seeds.rb`.